pkg/trace/sampler/config_watcher.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 The Go Authors. All rights reserved.
```

Comment:
Why is this Go authors?

Reply:
Wow nice catch thanks, I just copy pasted it from the wrong file it looks like

Reply:
fixed